server/store/subscriptions.go

```diff
@@ -198,3 +203,30 @@ func SubscriptionListFromJSON(bytes []byte) (*SubscriptionList, error) {
 	}
 	return subscriptionList, nil
 }
+
+func (s *Store) StoreSubscriptionChannelID(subscriptionID, webhookSecret, channelID string) error {
+	if err := s.StoreJSON(subscriptionID, SubscriptionWebhookSecretAndChannelMap{
+		webhookSecret: channelID,
+	}); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (s *Store) GetSubscriptionChannelID(subscriptionID string) (*SubscriptionWebhookSecretAndChannelMap, error) {
+	var storedWebhookSecret SubscriptionWebhookSecretAndChannelMap
+	if err := s.LoadJSON(subscriptionID, &storedWebhookSecret); err != nil {
+		return nil, err
+	}
+
+	return &storedWebhookSecret, nil
+}
```

Comment:
It seems off that the method `GetSubscriptionChannelID` is returning something called a webhook secret. The returned value is not a string either, which it seems like `secret` would be a string

Reply:
fixed